spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
@inukshuk, can I bother you for a minute? I was wondering if you could think of a better way to do this `expect(JSON_IS_VALID).to be true` test? I assume it would involve catching the `JSON::ParserError` exception and testing that, but I wasn't sure how to do that.